bazel/README.md

```diff
@@ -98,7 +98,9 @@ for how to update or override dependencies.
     ```
 
     ### Linux
-    On Linux, we recommend using the prebuilt Clang+LLVM package from [LLVM official site](http://releases.llvm.org/download.html).
+    On Linux, we recommend using the prebuilt Clang+LLVM package from [LLVM official site](http://releases.llvm.org/download.html) for Clang 14.
+    Note that newer versions of Clang do not include this package for Linux, but neither are newer versions of Clang able to build the Envoy dependencies.
```

Comment:
newer versions dont work for a bunch of reasons - there are some tarballs available - but the codebase needs to be updated before we can use them.

I would suggest just removing this bit

Reply:
Done - thanks!